stripe/test/resources/test_customers.py

```diff
@@ -234,20 +234,38 @@ def test_legacy_delete_subscription(self):
             None
         )
 
+    def test_list_customer_subscriptions(self):
+        customer = stripe.Customer.construct_from({
+            'id': 'cus_foo',
+            'subscriptions': {
+                'object': 'list',
+                'url': '/v1/subscriptions',
```

Comment:
I don't believe this is accurate. For operations on a subscription object, you are correct: it will use the new endpoint. However, for the operations on `ListObject` such as `create`, `all`, `retrieve`, it will [use the URL returned in the `url` parameter](https://github.com/stripe/stripe-python/blob/jacqueline-subs-v1/stripe/resource.py#L314). We aren't changing the subscriptions URL that's returned from the `/v1/customers/cus_foo` endpoint. 

{
  "id": "cus_7j89JgbvIey5zw",
  "object": "customer",
  "subscriptions": {
    "object": "list",
    "data": [
      {
        "id": "sub_7j8BvyTb5vfUef",
        "object": "subscription",
        ...
      }
    ],
    "has_more": false,
    "total_count": 1,
    "url": "/v1/customers/cus_7j89JgbvIey5zw/subscriptions"
  }
}
```